istio/datadog_checks/istio/istio.py

```diff
@@ -0,0 +1,134 @@
+# (C) Datadog, Inc. 2010-2018
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+
+from datadog_checks.checks.prometheus import Scraper, PrometheusCheck
+
+
+class Istio(PrometheusCheck):
+
+    def __init__(self, name, init_config, agentConfig, instances=None):
+        super(Istio, self).__init__(name, init_config, agentConfig, instances)
+        self.mixer_namespace = 'istio.mixer'
+        self.mesh_namespace = 'istio.mesh'
+        self.scrapers = {}
```

Comment:
nit: if this is not intended to be exposed to the caller can we rename it to `self._scrapers`?